File: src/weaponsprocurement/core/StockSellTarget.java
package weaponsprocurement.core;

import com.fs.starfarer.api.campaign.CargoAPI;
import com.fs.starfarer.api.campaign.econ.SubmarketAPI;

final class StockSellTarget {
    final SubmarketAPI submarket;
    final CargoAPI cargo;
    final int unitPrice;

    StockSellTarget(SubmarketAPI submarket, CargoAPI cargo, int unitPrice) {
        this.submarket = submarket;
        this.cargo = cargo;
        this.unitPrice = unitPrice;
    }
}

File: src/main/kotlin/weaponsprocurement/trade/plan/StockSellTarget.kt
package weaponsprocurement.trade.plan


import com.fs.starfarer.api.campaign.CargoAPI
import com.fs.starfarer.api.campaign.econ.SubmarketAPI

class StockSellTarget(
    @JvmField val submarket: SubmarketAPI,
    @JvmField val cargo: CargoAPI,
    @JvmField val unitPrice: Int,
)